game/CMakeLists.txt

```diff
@@ -132,6 +132,21 @@ foreach(lib SDL2 PangoCairo LibRSVG LibXML++ AVFormat SWResample SWScale ZLIB JP
 	add_definitions(${${lib}_DEFINITIONS})
 endforeach(lib)
 
+# Find all the optional libs, and #define HAVE_$lib if found (e.g.: HAVE_WebP)
```

Comment:
for other modules like opencv, webserver or midi we use the format `add_definitions("-DUSE_PORTMIDI")` and within code the `USE_PORTMIDI` ifdefs. Might be good to align that _if_ we want to keep it as an optional lib.

Personally since webp is such a big part of the web and libpng and jpeg are already as core libraries defined we might even think of adding this as one of the core libraries instead of making it optional. @Lord-Kamina what do ya think?